### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'route'

### ISSUE THREAD

**Issue by maxbitcoin** [secdev/scapy#290]
**[environment]**
python2.7.12
Ubuntu 16.04.1 LTS  32bit
PyCharm2016.2.1
Scapy  update on 2016.9.6

**[source code]**
```
# -*- coding:utf-8 -*-
from scapy.layers.inet import IP, TCP
from scapy.sendrecv import sr

def test():
    ans = sender('60.248.88.129', 80)
    print ans

def sender(dst, dport):
    ans = sr(IP(dst=dst) / TCP(dport=dport))
    return ans

if __name__ == "__main__":
    test()

```

**[error message]**
```
Traceback (most recent call last):
  File "/home/test/PycharmProjects/port/example/test.py", line 16, in <module>
    test()
  File "/home/test/PycharmProjects/port/example/test.py", line 7, in test
    ans = sender('60.248.88.129', 80)
  File "/home/test/PycharmProjects/port/example/test.py", line 12, in sender
    ans, unans = sr(IP(dst=dst) / TCP(dport=dport))
  File "/usr/local/lib/python2.7/dist-packages/scapy/sendrecv.py", line 339, in sr
    a,b=sndrcv(s,x,*args,**kargs)
  File "/usr/local/lib/python2.7/dist-packages/scapy/sendrecv.py", line 58, in sndrcv
    h = i.hashret()
  File "/usr/local/lib/python2.7/dist-packages/scapy/layers/inet.py", line 405, in hashret
    return strxor(inet_aton(self.src),inet_aton(self.dst))+struct.pack("B",self.proto)+self.payload.hashret()
  File "/usr/local/lib/python2.7/dist-packages/scapy/packet.py", line 194, in __getattr__
    return fld.i2h(self, v)
  File "/usr/local/lib/python2.7/dist-packages/scapy/fields.py", line 276, in i2h
    iff,x,gw = conf.route.route(dst)
AttributeError: 'NoneType' object has no attribute 'route'
```

**[try]**

If I use   ```from scapy.all import *```  , there is no error ,but I can not set break point in PyCharm to debug it.

**Comment by p-l-** [comment]
Hi,

Before anything, can you please format your comment? It's unreadable for now, sorry.

**Comment by robin-jarry** [comment]
I guess this issue is fixed :)

**Comment by p-l-** [comment]
Fixed in #292 by @robin-jarry.
